.gitignore

```diff
@@ -61,4 +61,5 @@ __pycache__
 /tools/mo/*.svg
 /src/plugins/intel_cpu/tools/commit_slider/*.json
 /src/plugins/intel_cpu/tools/commit_slider/slider_cache/*
+/src/plugins/intel_cpu/thirdparty/ComputeLibrary/build/libarm_compute-static.a
```

Comment:
Do we really need to put this library in thirdparty? 
Cannot it be in common binaries?

Reply:
Because of changes in scons config, static library is build in src directory and @ilya-lavrenov asked me to add static file location into gitignore

Reply:
Shouln't we add whole build folder to gitignore then?

Reply:
fixed